games/Kingdom Hearts 2.yaml

```
@@ -0,0 +1,81 @@
+Kingdom Hearts 2:
+  LevelDepth:
+    level_50: 50
+    level_50_sanity: 50
+  Sora_Level_EXP:
+    5: 50
+  Valor_Form_EXP:
+    10: 50
+  Wisdom_Form_EXP:
+    4: 50
+  Limit_Form_EXP:
+    4: 50
+  Master_Form_EXP:
+    4: 50
+  Final_Form_EXP:
+    4: 50
+  Summon_EXP:
+    4: 50
+  Schmovement:
+    level_1: 50
+  RandomGrowth:
+    0: 50
+  Promise_Charm:
+    'true': 50
+  Goal:
+    three_proofs: 25
+    lucky_emblem_hunt: 25
+  FinalXemnas:
+    'true': 50
+  LuckyEmblemsAmount:
+    30: 50
```

```suggestion
    random-range-30-35: 50
```
can cut this down by 5 rows